lib/render.js

```diff
@@ -1,9 +1,43 @@
 var hljs = require("highlight.js")
-var marked = module.exports = require("marked")
+var MarkdownIt = require("markdown-it")
 
-marked.setOptions({
+var md = new MarkdownIt({
+  html: true,
   langPrefix: "hljs ",
   highlight: function(code) {
     return hljs.highlightAuto(code).value;
   }
 })
+
+module.exports = function render(markdown) {
+  // return require('marked')(markdown)
+  return md.render(markdown)
+}
+
+var defaultRender = md.renderer.rules.heading_open
```

Comment:
I think, here is no need to keep default renderer, because it will be called only for empty headers. Header always generate this sequeance: `header_open` + `inline` + `header_close`, and i can't imagine situation when this can be changed.